app/views/support/Pricing.scala

```diff
@@ -62,7 +62,17 @@ object Pricing {
       }
     }
 
+    def headlinePricingForDiscountedPeriod[M[+A], A <: LandingPage](discountPromo: Promotion[PercentDiscount, M, A], currency: Currency) = {
+      val originalAmount = unsafePrice(currency)
+      val discountAmount = discountPromo.promotionType.applyDiscount(originalAmount, plan.billingPeriod)
+      discountPromo.promotionType.durationMonths.fold {
```

Comment:
"From" could just be made an Option[String] with .mkString used in the s"..." so you don't have to repeat the full String template, or use a .fold on durationMonths